src/io/dsv.rs

```diff
@@ -29,13 +29,13 @@
 //! let table_reader: Box<dyn TableReader> = Box::new(csv_reader);
 //! ```
 //!
-//! The example first creates a [`DSVReader`] which uses comma as its separator.
-//! Then it packs it as a trait object of type [`TableReader`], which represents the visibility for the phyical layer.
+//! The example first creates a [`DSVReader`] which uses a comma as its separator.
+//! It passes the reader to the physical layer as a [`TableReader`] object.
 //!
 //! ## Physical layer
-//! The physical layer receives a trait object [`TableReader`], containing
+//! The physical layer receives a trait object [`TableReader`], which is
 //! the instantiated `csv_reader` from the example above.
-//! For this example the same variable is used.
+//! The same variable is used.
```

Comment:
_Who_ uses the same variable? What even is this variable – if it's the trait object, we already said that it's the one from the example above?